### PR TITLE
update the nova policy file with correct permissions

### PR DESCRIPTION
**Pull request by lihkin213**
The following changes are in this PR.

- Pick up some of the updated to the nova in-code policy.
- Simplified and removed some of the rules as they were redundant. 
- Some access rules around servers were too restrictive and did not match the default community permissions for the members, for example server start  and stop (and other actions) were restricted only to the admin, cloud_admin, project_admin and the user that created the resource. These actions should be allowed by any member of the project.
- Some access rules were too lenient that needed to be restricted to the owner of the resource and, admin and cloud_admin. For example, all rules for os-keypairs and keypairs, that allowed access to them by admins as well as other members of the project.

### CODE REVIEW

File: roles/nova-common/templates/etc/nova/policy.json [hunk 1]
```diff
@@ -36,8 +32,8 @@
     "compute:get_diagnostics": "rule:admin_or_owner",
     "compute:get_instance_diagnostics": "rule:admin_or_owner",
 
-    "compute:start": "rule:admin_or_projAdmin_or_cloudAdmin_or_owner",
-    "compute:stop": "rule:admin_or_projAdmin_or_cloudAdmin_or_owner",
+    "compute:start": "rule:admin_or_owner",
+    "compute:stop": "rule:admin__owner",
```
Comment (nirajdp76) [inline roles/nova-common/templates/etc/nova/policy.json]: should be admin_or_owner

**Comment by lihkin213** [comment]
retry

**Comment by lihkin213** [comment]
retest

**Comment by nirajdp76** [comment]
LGTM

**Comment by lihkin213** [comment]
retest

**Pull request merged**